### ISSUE TITLE
Are there spammers using your banner?

### ISSUE THREAD

**Issue by errael** [vshymanskyy/StandWithUkraine#115]
Take a look at https://github.com/vim/vim/wiki

Looks like the same banner, but goes to a different URL, with a slightly modified name. 

Any idea if it's legitimate?

A question was asked in
https://groups.google.com/u/2/g/vim_dev/c/0bjLrv9494s
about the change to the vim wiki.

**Comment by vshymanskyy** [comment]
currently don't see anything there

**Comment by cristaloleg** [comment]
@vshymanskyy see history https://github.com/vim/vim/wiki/Home/_history

**Comment by vshymanskyy** [comment]
https://standwithukraine.pp.ua/ links to https://www.comebackalive.in.ua/, which is not that bad.
The site is legit, however not directly associated with this repo.

**Comment by vshymanskyy** [comment]
@errael after some verification, we consider `standwithukraine.pp.ua` as **possible a phishing attempt**.
ComeBackAlive org refused any relation to this website.
However, it looks it was just used once in the VIM wiki. Currently, it's not mentioned anywhere else: [seash results](https://github.com/vshymanskyy/StandWithUkraine/search?q=standwithukraine.pp.ua)

**Comment by errael** [comment]
@vshymanskyy , thanks for the info. Yes, I did get to your link a few days ago, and mentioned on the vim mailing list that you looked legit.

The biggest question I've always got with donations, even to legit organizations, is how much goes to assistance (and not to bureaucracy).

Your efforts are appreciated.

**Comment by webknjaz** [comment]
> The biggest question I've always got with donations, even to legit organizations, is how much goes to assistance (and not to bureaucracy).

TL;DR since you care about the bureaucracy, don't send money to the international red cross — there's been countless of confirmations that they are doing next to nothing. As for Back Alive — they have a good track record of supporting the Ukrainian defenders throughout the previous 8 years and I trust that everything gets converted into the help for the army, they're known to be one of the most effective supporters (in their field). Just note that they can't buy/supply weapons. If you specifically want to contribute to that — it's probably best to send the money to the official account of the national bank or the embassies/consulates of Ukraine in your country — these are the institutions that can legally aquire weapons.
